app/tasks/bookmark_save_task.py

```
@@ -1,229 +1,379 @@
+        chunk_vectors = await vector_service.save_document(
+            session=session,
+            user_id=bookmark_data.user_id,
+            source_id=f"{bookmark_data.star_id}_chunk_{chunk_data['chunk_index']}",
+            source_type="bookmark_chunk",
+            content=chunk_data["content"],
+            title=bookmark_data.title,
+            url=bookmark_data.url,
+            keywords=bookmark_data.keywords,
+            summary=bookmark_data.summary,
+            extra_metadata=chunk_metadata
+        )
```

Pylint (3.3.7)

[error] No value for argument 'source_data' in method call

(E1120)